### PR TITLE
Add gpcam, fvgp and hgdl

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/gpcam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - wheel
```
Comment (ocefpaf) [inline recipes/gpcam/meta.yaml:25]: Wheel is probably not needed.

File: recipes/hgdl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - wheel
+    - versioneer
```
Comment (ocefpaf) [inline recipes/hgdl/meta.yaml:24]: I see wheel and versioneer in the run deps for these packages but I believe those are not needed Maybe upstream added them by mistake?

Comment (jan-janssen) [reply recipes/hgdl/meta.yaml:24]: Versioneer is used in the `setup.py` script. https://github.com/lbl-camera/fvGP/blob/master/setup.py

File: recipes/gpcam/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipes/gpcam/meta.yaml]: @ocefpaf I added the required patch to remove it from the dependencies.

File: recipes/hgdl/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/hgdl/meta.yaml]: @ocefpaf I added the required patch to remove it from the dependencies.

**Comment by jan-janssen** [comment]
@ocefpaf Can you take another look?

File: recipes/gpcam/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/gpcam/meta.yaml]: If you can, file an issue upstream too.

**Pull request merged**